package-lock.json

```diff
@@ -130,6 +130,12 @@
 				}
 			}
 		},
+		"@zoltu/typescript-transformer-append-js-extension": {
```

Comment:
I think you need to `npm install` again, or perhaps your working tree is dirty.  This package and `ttypescript` are both added in my branch, but not here.  From this and some other comments you recently made in chat, I _think_ you somehow have a dirty checkout.